static/sass/components/_author.scss

```diff
@@ -0,0 +1,108 @@
+// Author Box
+// ==========
+
+[data-module] {
+  + [data-module='rst-author'] {
```

Comment:
Why can't we target this directly, without the sibling selector? Is `rst-author` used somewhere else?